internal/commands/builder_inspect.go

```diff
@@ -0,0 +1,78 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/builder"
+	"github.com/buildpacks/pack/internal/builder/writer"
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+type BuilderInspector interface {
+	InspectBuilder(name string, daemon bool, modifiers ...pack.BuilderInspectionModifier) (*pack.BuilderInfo, error)
+}
+
+type BuilderInspectFlags struct {
+	Depth        int
+	OutputFormat string
+}
+
+func BuilderInspect(logger logging.Logger,
+	cfg config.Config,
+	inspector BuilderInspector,
+	writerFactory writer.BuilderWriterFactory,
+) *cobra.Command {
+	var flags BuilderInspectFlags
+	cmd := &cobra.Command{
+		Use:     "inspect <builder-image-name>",
+		Args:    cobra.MaximumNArgs(1),
+		Aliases: []string{"inspect-builder"},
+		Short:   "Show information about a builder",
+		Example: "pack builder inspect cnbs/sample-builder:bionic",
+		Long:    "Show information about the builder provided. If no argument is provided, it will inspect the default builder, if one has been set.",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			imageName := cfg.DefaultBuilder
+			if len(args) >= 1 {
+				imageName = args[0]
+			}
```

Comment:
This was previously an undocumented feature. I've added it to the long description of the command now.